### PR TITLE
several updates

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-2bcd7ae3`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-d9b6c71e`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-b317de31`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-3725c20d`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-82ba7704`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-3cd98143`

File: content/sre/argocd/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Introduction
+
+We use GitOps to continuously deliver application changes
+
+[Argo CD](https://argoproj.github.io/argo-cd/){: target=_blank} is a declarative, GitOps continuous delivery tool for Kubernetes. The deployment environment is a namespace in a container platform.
```
Comment (usamaahmadkhan) [inline content/sre/argocd/01-introduction.md]: {: target=_blank} doesn't do anything

File: content/sre/code-quality/01-introduction.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Introduction
+
+[SonarQube](https://www.sonarqube.org/) 
+
+We SonarQube to analyze your code's quality
```
Comment (usamaahmadkhan) [inline content/sre/code-quality/01-introduction.md]: * We use Sonarqube

File: content/sre/code-quality/01-introduction.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+**From a software architect’s standpoint**, SonarQube is worth the time because it helps you keep an eye on whether your cleanly delineated initial design is being degraded over time with creeping dependency cycles. It can show you whether the internal coding rules are being followed, and it can help you spot rising complexity that needs to be refactored.
+
+**From a project management standpoint**, SonarQube is worth the focus because testing alone isn’t enough. It can only show whether software does what it’s supposed to do: its level of external quality. On the other hand, SonarQube analyzes and fosters internal quality: whether an application will run optimally and be readily maintainable and extensible down the road.
+
+**From a business standpoint**, SonarQube offers a strong ROI because its acquisition and setup costs are low, and its intuitive interfaces mean that very little training is required. Add to that the fact that its adoption within an organization is typically viral, and you’ve got a minimal investment that produces what quickly become significant results.
```
Comment (usamaahmadkhan) [inline content/sre/code-quality/01-introduction.md]: We are offering it as a managed add-on, should we talk about costs?

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-7c152531`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-4aec1b3f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-175-1c343eae`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.116`

**Pull request merged**